### ISSUE TITLE
JSON format for power management

### ISSUE THREAD

**Issue by tpatki** [flux-framework/rfc#223]
Create an RFC for standardized vendor-neutral interaction with libraries such as variorum to interact with Flux, and to allow for interfacing with other power management libraries/APIs such as PowerAPI or PlatformIO should they choose to implement a JSON interface to Flux.

**Comment by dongahn** [comment]
Thank you for opening this up, @tpatki!

**Comment by tpatki** [comment]
@dongahn: Is there a guideline on how to write RFCs? Should I just follow the format of the other RFCs we have under Flux? I've never written an RFC, so making sure the first cut I have is in the correct direction. 

Once we have a first cut, I am assuming the Flux team can iterate over it. And once we are in agreement, @slabasan and I will also iterate over these in the PowerStack working group meetings.

**Comment by dongahn** [comment]
@tpatki: I don't think we have RFC for RFC :-). Modeling after other RFCs make sense. @grondo or @garlick?

**Comment by grondo** [comment]
Yes, just follow an existing example. Be sure to use the language definitions from ietf [RFC 2119](https://tools.ietf.org/html/rfc2119) (as the other RFCs do). The process for developing an RFC is the same as other PRs ([C4.1](https://github.com/flux-framework/rfc/blob/master/spec_1.adoc), with modifications noted at the bottom of [rfc/README](https://github.com/flux-framework/rfc/blob/master/README.adoc))
